nnpdf_data/nnpdf_data/new_commondata/STAR_2012_1JET_510GEV/metadata.yaml

```diff
@@ -0,0 +1,57 @@
+setname: "STAR_2012_1JET_510GEV"
+
+nnpdf_metadata:
+  experiment: "STAR"
+  nnpdf31_process: "JETS"
+
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1493842"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1493842"
+  version: 1
+arXiv:
+  url: https://arxiv.org/abs/1906.02740
```

Comment:
@giacomomagni, @toonhasenack: Are these metadata correct??

I am instead seeing this: https://www.hepdata.net/record/ins1738738 with a date 2019.

Reply:
yes they are clearly wrong, thanks for spotting

Reply:
But then if so, shouldn't the dataset name also be updated with the correct date?

Reply:
data were collected in 2012 for the paper?

Reply:
That's right! All good then.